src/main/java/net/rptools/maptool/client/ui/zone/renderer/ZoneRenderer.java

```diff
@@ -85,20 +85,17 @@ public class ZoneRenderer extends JComponent
 
   private static final long serialVersionUID = 3832897780066104884L;
   private static final Logger log = LogManager.getLogger(ZoneRenderer.class);
-
-  private static final Color TRANSLUCENT_YELLOW =
-      new Color(Color.yellow.getRed(), Color.yellow.getGreen(), Color.yellow.getBlue(), 50);
+  protected static final ZoneRendererConstants constants = new ZoneRendererConstants();
```

Comment:
No need to instantiate `ZoneRendererConstants`, it can be used statically where needed.